src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -190,22 +341,48 @@ def from_dict(cls, export_id, ex_dict):
                    ex_dict['access_type'],
                    [Client.from_dict(client) for client in ex_dict['clients']])
 
+    def to_dict(self):
+        return {
+            'export_id': self.export_id,
+            'path': self.path,
+            'cluster_id': self.cluster_id,
+            'pseudo': self.pseudo,
+            'access_type': self.access_type,
+            'squash': self.squash,
+            'security_label': self.security_label,
+            'protocols': sorted([p for p in self.protocols]),
+            'transports': sorted([t for t in self.transports]),
+            'fsal': self.fsal.to_dict(),
+            'clients': [client.to_dict() for client in self.clients]
+        }
+
+
 class FSExport(object):
     def __init__(self, mgr, namespace=None):
         self.mgr = mgr
         self.rados_pool = 'nfs-ganesha'
-        self.rados_namespace = namespace #TODO check if cluster exists
-        self.export_conf_blocks = []
+        self.rados_namespace = namespace
         self.exports = {}
 
-    def check_fs(self, fs_name):
-        fs_map = self.mgr.get('fs_map')
-        return fs_name in [fs['mdsmap']['fs_name'] for fs in fs_map['filesystems']]
+        try:
+            log.info("Begin export parsing")
+            for cluster_id in available_clusters(self.mgr):
+                self.export_conf_objs = []  # type: List[Export]
+                self._read_raw_config(cluster_id)
+                self.exports[cluster_id] = self.export_conf_objs
+            log.info(f"Exports parsed successfully {self.exports.items()}")
+        except orchestrator.NoOrchestrator:
+            # Pass it for vstart
+            log.info("No orchestrator error")
```

Comment:
> Ambiguous message. Something less ambiguous. `orchestrator not found` is better.
> 
> I don't follow why we catch this error for vstart?
> 

Because in vstart dashboard module is started first and then cephadm gets enabled. 

> This would also catch the error for non-vstart deployments. Is that OK?

It is fine as if any of the commands are issued then orchestrator error would be raised